app/src/main/java/org/mozilla/fenix/browser/BrowserAnimator.kt

```diff
@@ -38,11 +37,8 @@ class BrowserAnimator(
         get() = swipeRefresh.get()
 
     fun beginAnimateInIfNecessary() {
-        viewLifecycleScope.get()?.launch {
-            delay(ANIMATION_DELAY)
-            unwrappedEngineView?.asView()?.visibility = View.VISIBLE
-            unwrappedSwipeRefresh?.background = null
-        }
+        unwrappedEngineView?.asView()?.visibility = View.VISIBLE
```

Comment:
When `coverUntilFirstPaint` was introduced I asked Agi about those visibility changes and I think he mentioned that we can get rid of them now too. Maybe we could test this and remove this in a follow-up in Fenix and AC? :)

Reply:
I will investigate! In Fenix I know we also toggle the engine view visiblity when we `captureEngineViewAndDrawStatically` when navigating to Settings for example and then return to the browser.